_includes/rest/objects.md

```diff
@@ -223,14 +225,16 @@ print result
 </code></pre>
 </div>
 
-The response body is a JSON object containing just an `updatedAt` field with the timestamp of the update.
+The response body is a JSON object containing by default an `updatedAt` field with the timestamp of the update.
 
 ```json
 {
   "updatedAt": "2011-08-21T18:02:52.248Z"
 }
 ```
 
+The response body also includes any field that was modified using `beforeSave` and `afterSave` triggers.
```

Comment:
Only the case if the `afterSave` trigger has completed - if there is a long running operation new changes may not be included.

Reply:
same thing here